### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Philip-21**
**Description**

This PR fixes #
Pull Request for Utils package Documentation  #286 
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: utils/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## The Utils Package 
+ The Utils Package is the Central component of  Meshkit. This package encompases different packages that offers a simplified and tailored experience for developers working with Kubernetes and Helm. Developers can easily interact with Kubernetes and Helm without relying on the command line versions by utilizing higher-level functions from the utils package. 
+ 
+ Here is a brief description of some of the packages embedded in Utils:
+
+
+ ### Kuberentes
```
Comment (Aisuko) [inline utils/readme.md:7]: Does this line work well? Why add a blank before it?

File: utils/readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  This package contains certain packages such as describe, expose, Kompose, manifests  and walker . Below is a description of each package : 
+
+##### Describe 
+  Describe Package  is a comprehensive and user-friendly solution for describing Kubernetes objects through the Kubernetes API. With its rich set of functionalities, it contains components for creating and initiaizing a meshclient, allowing users to retrieve information about various Kubernetes resources such as pods, deployments, jobs, services, and more to interact with the kubernetes Api.
+  Overall, the describe package provides a convenient way to retrieve detailed information about Kubernetes resources in a standardized format.  
+ 
+
+ ##### Expose
```
Comment (Aisuko) [inline utils/readme.md:15]: Same

File: utils/readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ 
+
+ ##### Expose 
+ - The Expose Package provides functionality for exposing Kubernetes resources as services.
+ - The package imports various packages from the standard library and external dependencies. 
+ - It contains fields for specifying the service type, load balancer IP, cluster IP, namespace,  session affinity, name, annotations, and a logger. 
+ - The <code>Expose()</code> is the main function of the package this takes a Kubernetes clientset, REST config, Config object, and a list of resources to expose.
+ - It uses a Traverser to iterate over the resources and generate the corresponding services.
+ Overall, this package provides a way to expose Kubernetes resources as services with customizable configurations.
+
```
Comment (Aisuko) [inline utils/readme.md:22]: Why there are so many blanks here?

**Review by Aisuko — CHANGES_REQUESTED** [review]

File: utils/readme.md (hunk 1 above)
Comment (Philip-21) [reply utils/readme.md:7]: The aspects where i left spaces are where i will add descriptions for other packages , i  just pushed this pr for reviews on the explanations i've made.

Comment (Aisuko) [reply utils/readme.md:7]: I am not sure I got your point, but please follow the syntax of markdown.

Comment (Philip-21) [reply utils/readme.md]: Alright , 
from this  commit i just pushed i think i have made adjustment to the  spaces .

**Comment by Philip-21** [comment]
@Aisuko i have made adjustments on the spaces and blanks . 
This is all i could come up with for the docs of the utils package.

**Comment by Philip-21** [comment]
> @Philip-21 💪

Thanks @leecalcote

**Pull request merged**